File: base/src/main/java/com/windscribe/vpn/api/response/XPressLoginVerifyResponse.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.vpn.api.response;

import androidx.annotation.NonNull;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class XPressLoginVerifyResponse {

    @SerializedName("session_auth_hash")
    @Expose
    private String sessionAuth;

    public String getSessionAuth() {
        return sessionAuth;
    }

    @NonNull
    @Override
    public String toString() {
        return "XPressLoginVerifyResponse{" +
                "sessionAuth='" + sessionAuth + '\'' +
                '}';
    }
}

File: base/src/main/java/com/windscribe/vpn/api/response/XPressLoginVerifyResponse.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.vpn.api.response

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

@Keep
class XPressLoginVerifyResponse {
    @SerializedName("session_auth_hash")
    @Expose
    val sessionAuth: String? = null

    override fun toString(): String =
        "XPressLoginVerifyResponse{" +
            "sessionAuth='" + sessionAuth + '\'' +
            '}'
}
